tests/transforms/test_transforms.py

```diff
@@ -38,6 +45,14 @@ def test_scale_pointcloud(device='cpu'):
     assert_allclose(pc_, torch.ones(4, 3))
 
 
+def test_translate_pointcloud(device='cpu'):
```

Comment:
can you test more variation of dimensions ? (2 and 3)

Reply:
Added more tests in next commit :D